src/components/checkbox/checkbox.style.ts

```diff
@@ -10,11 +9,20 @@ import StyledLabel, {
   StyledLabelContainer,
 } from "../../__internal__/label/label.style";
 import StyledValidationIcon from "../../__internal__/validations/validation-icon.style";
-import StyledFormField from "../../__internal__/form-field/form-field.style";
-import StyledIcon from "../icon/icon.style";
 import baseTheme from "../../style/themes/base";
-
-const CheckboxStyle = styled.div`
+import { ValidationProps } from "../../__internal__/validations";
+
+interface CheckboxStyleProps extends ValidationProps, MarginProps {
```

Comment:
```suggestion
interface StyledCheckboxProps extends ValidationProps, MarginProps {
```
In my opinion I think this would be good for consistency with other components, though this is minor and happy to accept otherwise

Reply:
I agree, I'll change that